stackslib/src/config/mod.rs

```diff
@@ -4047,9 +4153,101 @@ impl AtlasConfigFile {
 #[derive(Clone, Deserialize, Default, Debug, Hash, PartialEq, Eq, PartialOrd)]
 #[serde(deny_unknown_fields)]
 pub struct EventObserverConfigFile {
+    /// URL endpoint (hostname and port) where event notifications will be sent via HTTP POST requests.
+    ///
+    /// The node will automatically prepend `http://` to this endpoint and append the
+    /// specific event path (e.g., `/new_block`, `/new_mempool_tx`).
+    /// Therefore, this value should be specified as `hostname:port` (e.g., "localhost:3700").
+    ///
+    /// **Do NOT include the `http://` scheme in this configuration value.**
+    ///
+    /// This should point to a service capable of receiving and processing Stacks event data.
+    ///
+    /// Default: No default. This field is required.
     pub endpoint: String,
+    /// List of event types that this observer wants to receive.
```

Comment:
```suggestion
    /// List of event types that this observer is configured to receive.
```

Reply:
hmm. i don't think we should link to markdown from the source code personally. 
if either changes (or even the location changes/files renamed) - it creates tech debt. 

maybe a line like "check the event-dispatcher docs in `./docs`" or something generic ?

Reply:
otherwise, i would say it's fine as-is unless you want to add the above.